lib/splash.js

```diff
@@ -15,8 +15,7 @@ export default function splash ({ path, name, org, coc, logo, active, total, cha
     dom('p.status',
       active
         ? [
-          dom('b.active', active), ' users online now of ',
-          dom('b.total', total), ' registered.'
+          dom('b.active', active), ' users online now.'
         ]
         : [dom('b.total', total), ' users are registered so far.']
     ),
```

Comment:
Should this probably be:

dom('p.status',
      active
        &&  [
          dom('b.active', active), ' users online now.'
        ]
    ),

I'm not sure what would happen if !active, so I would want to test this out